src/gettime.h

```diff
@@ -19,7 +19,6 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #pragma once
 
-#include "irrlichttypes.h"
```

Comment:
I made this change when the chat message was formatted in the C++-side.

I haven't removed this, as this header needn't be included in this file and would increase compilation times for no reason.